adagrams/game.py

```diff
@@ -1,13 +1,107 @@
+import random
 from random import randint
 
+def establish_pool():
+    pool = []
+    letter_counts= {
+        'A': 9, 
+        'B': 2, 
+        'C': 2, 
+        'D': 4, 
+        'E': 12, 
+        'F': 2, 
+        'G': 3, 
+        'H': 2, 
+        'I': 9, 
+        'J': 1, 
+        'K': 1, 
+        'L': 4, 
+        'M': 2, 
+        'N': 6, 
+        'O': 8, 
+        'P': 2, 
+        'Q': 1, 
+        'R': 6, 
+        'S': 4, 
+        'T': 6, 
+        'U': 4, 
+        'V': 2, 
+        'W': 2, 
+        'X': 1, 
+        'Y': 2, 
+        'Z': 1
+    }
+  
+    for key, value in letter_counts.items():
+        for i in range(value):
+            pool.append(key)
+    return pool
+
 def draw_letters():
-    pass
+    pool = establish_pool()
+    pool_size = len(pool)
+    hand_cards = []
+    # pull the cards 10 times, decreasing pool size by one by pop the card out of the pool. 
+    for i in range(10):
+        rand_index = random.randint(0,pool_size-1)
+        hand_cards.append(pool[rand_index])
+        pool.pop(rand_index)
+        pool_size -= 1 # or len(pool)
+    pool += hand_cards
+    return hand_cards
 
 def uses_available_letters(word, letter_bank):
-    pass
+    word = word.upper()
+    letter_dict = {}
+    for letter in letter_bank:
+        # default letter_dict[letter] = 0, using the .get() method 
+        # and add one each time encounter the letter, the first encouter also adds 1 
+        #it's probably also workable to build a hashtable of every letter possible and count every letter into the dictionary 
+        letter_dict[letter] = letter_dict.get(letter, 0) + 1
+    for letter in word: 
```

Comment:
Consider adding a few blank lines within your function to separate logically distinct sections of the code (e.g., setup, processing, return).  According to [PEP 8](https://peps.python.org/pep-0008/#blank-lines), blank lines can improve readability by visually grouping related operations. 

```suggestion
        letter_dict[letter] = letter_dict.get(letter, 0) + 1
        
    for letter in word: 
```